mono/mini/TestDriver.cs

```diff
@@ -13,10 +13,40 @@ public class CategoryAttribute : Attribute
 		get; set;
 	}
 }
+public class TestDriverReporter
+{
+	private int failedTests = 0;
+	private int skippedTests = 0;
+	private int executedTests = 0;
+
+	public int FailedTests {
+		get {
+			return failedTests;
+		}
+	}
+
+	public int SkippedTests {
+		get {
+			return skippedTests;
+		}
+	}
+
+	public int ExecutedTests {
+		get {
+			return executedTests;
+		}
+	}
```

Comment:
The JIT will inline the property access. Also it's not like this code is anywhere near a hot path 😄